agent/src/util/crictl_containers.rs

```diff
@@ -127,8 +133,8 @@ mod tests {
             expected_2,
             get_container_slot_usage(&format!(
                 "{{ \"containers\": [ {}, {} ] }}",
-                &get_container_str("\"akri.agent.slot\": \"foo1\","),
-                &get_container_str("\"akri.agent.slot\": \"foo2\","),
+                &get_container_str("\"akri.agent.slot-foo1\": \"foo1\","),
+                &get_container_str("\"akri.agent.slot-foo2\": \"foo2\","),
```

Comment:
@johnsonshih, looks like this needed the changes from https://github.com/project-akri/akri/pull/560 which is now merged. Just a bump to rebase so only config resource changes are here